databaseconnect.py

```diff
@@ -177,53 +169,53 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    #cur.execute(f"UPDATE statement_table SET sentence='{H}' WHERE id={last_id}")
+    #cur.execute("UPDATE statement_table SET sentence={H} WHERE id={last_id}")
     cur.execute("UPDATE statement_table SET sentence = '{}' WHERE id = '{}'".format(H, last_id))
     db.commit()
     B = "Thank you! I have learnt this."
     return B, 0
 
-
+@logger_config.logger
 def clear_table(table_name):
     db = connection_to_database()
     cur = db.cursor()
 
     if table_name in ("question_table", "statement_table"):
         tables_to_be_cleaned = ("question_table", "statement_table")
-        print("The following tables will be cleaned:\n")
+        cur.execute("The following tables will be cleaned:\n")
         for table in tables_to_be_cleaned:
             describe_table(cur, table)
 
         if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y", "y"):
             for table in tables_to_be_cleaned:
-                cur.execute(f"DELETE FROM {table}")
+                cur.execute("DELETE FROM {table}")
             db.commit()
-            print("Tables cleaned successfully")
+            cur.execute("Tables cleaned successfully")
```

Comment:
Do not replace `print` statements with `cur.execute`

Reply:
Resolved 👍